components/org.wso2.carbon.siddhi.editor.core/src/main/resources/web/js/design-view/drop-elements.js

```diff
@@ -1663,17 +1666,17 @@ define(['require', 'log', 'lodash', 'jquery', 'partition', 'stream', 'query', 'f
                 * update the other elements data connected to current element. ex: when a stream is deleted from a
                 * query, from clause in the query will be updated as undefined.
                 * */
-                setTimeout(function () {
-                    var outConnections = self.jsPlumbInstance.getConnections({source: elementId + '-out'});
-                    var inConnections = self.jsPlumbInstance.getConnections({target: elementId + '-in'});
-
-                    _.forEach(outConnections, function (connection) {
-                        self.jsPlumbInstance.deleteConnection(connection);
-                    });
-                    _.forEach(inConnections, function (connection) {
-                        self.jsPlumbInstance.deleteConnection(connection);
-                    });
-                }, 100);
+                // setTimeout(function () {
```

Comment:
shall we verify whether we need this and remove if not needed

Reply:
I think it can be removed since it doesn't make any effect while deleting an element

Reply:
fixed in d389caca3be2b4cb8e562e6c5e03e391b07fd6c2